### PR TITLE
Serve html

### PR DESCRIPTION
**Pull request by matejkrajcovic**
- Backend no longer serves json responses but html files using jinja2 template engine.
- Both searching by name and by age are supported.
- Example data of four persons were added (Elon Musk, Steve Jobs, Nikola Tesla and Mark Zuckerberg).

### CODE REVIEW

**Comment by matejkrajcovic** [comment]
@mrshu @pepol @ViRPo

**Comment by matejkrajcovic** [comment]
@pepol @mrshu @ViRPo

File: README.md [hunk 1]
```diff
@@ -14,7 +15,8 @@ $ python app.py
 	- config.py # Configuration for development and testing environment
 	- models.py # SQLAlchemy models
 	- views.py # Flask routes
+    - search.py # helper functions to search
 - test
-	- __init__.py # Base testcase to prepare and destroy application and database
+	- \_\_init\_\_.py # Base testcase to prepare and destroy application and database
```
Comment (pepol) [inline README.md:20]: If you want to quote code in markdown, you can put the thing between backticks (`) just like `this`. See [GitHub's guide to markdown](https://guides.github.com/features/mastering-markdown/) for more.

Comment (matejkrajcovic) [inline README.md:20]: I don't want to quote it, just prevent making it bold: **init**.py vs. __init__.py.

Comment (pepol) [inline README.md:20]: oh, okay then...

**Pull request merged**